### PR TITLE
#147 Readme update

### PR DESCRIPTION
**Pull request by vryazanov**
#147 
* Added instruction to setup PostgreSQL for Debian / Ubuntu.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#151](https://github.com/timelesslounge/timelessis/issues/151) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#151](https://github.com/timelesslounge/timelessis/issues/151) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/fda3f02b-75bc-40a1-a0c4-218b43251a3b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -197,8 +197,25 @@ cd timelessis
 pip install -r requirements.txt
 ```
 
-### Database Setup
+### Setup PostgreSQL for Debian / Ubuntu
+Debian / Ubuntu repositories include PostgreSQL by default. To install PostgreSQL, use the apt-get (or other apt-driving) command:
+```
+apt-get update
```
Comment (vladarefiev) [inline README.md]: I think we should add `sudo` here, WDYT?

Comment (vryazanov) [reply README.md]: I just tried these commands under root user in docker container. but sounds reasonable. I will fix it

**Comment by vladarefiev** [comment]
@vryazanov everything is good for me, only one comment

File: README.md [hunk 2]
```diff
@@ -197,8 +197,25 @@ cd timelessis
...
+### Setup PostgreSQL for Debian / Ubuntu
+Debian / Ubuntu repositories include PostgreSQL by default. To install PostgreSQL, use the apt-get (or other apt-driving) command:
+```
+sudo apt-get update
+sudo apt-get install postgresql-10
+```
+If packages are not found, do the following steps:
+1. Create the file /etc/apt/sources.list.d/pgdg.list and add a line for the repository:
+```
+deb http://apt.postgresql.org/pub/repos/apt/ {DEBIAN_OR_UBUNTU_VERSION}-pgdg main
```
Comment (vladarefiev) [inline README.md]: @vryazanov I think the same here?

Comment (vryazanov) [reply README.md]: @vladarefiev fixed

**Comment by vladarefiev** [comment]
@rultor good to merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#151](https://github.com/timelesslounge/timelessis/issues/151) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka Confirm

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/timelesslounge/timelessis/issues/151#issuecomment-461873195))

@ypshenychka There is no quality review for [#151](https://github.com/timelesslounge/timelessis/issues/151), no performer

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**